magda-search-api-node/src/createApiRouter.ts

```diff
@@ -0,0 +1,216 @@
+import * as express from "express";
+import * as _ from "lodash";
+import buildJwt from "@magda/typescript-common/dist/session/buildJwt";
+import * as joi from "joi";
```

Comment:
It seems `joi` has been moved to `@hapi/joi`. If it's easier, probably just update the dependency name.
> This module has moved and is now available at @hapi/joi. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.